Dockerfile

```diff
@@ -28,13 +23,17 @@ ENV C_INCLUDE_PATH=/usr/include/gdal
 RUN apt-get install -y binutils libproj-dev gdal-bin libjpeg-dev
 
 # Run the application:
-COPY disasterinfosite /disasterinfosite
-WORKDIR /disasterinfosite
+COPY . /app
```

Comment:
This command copies everything, including the disasterinfosite, so we don't need to copy it separately.

Reply:
oh right. I missed the . somehow. Thanks.